### PR TITLE
Add mail4qa.com

### PR DESCRIPTION
**Pull request by reporty122**
https://console.mail4qa.com/inbox/

### CODE REVIEW

File: list.txt [hunk 1]
```diff
@@ -55737,4 +55738,4 @@ zzuwnakb.pl
 zzv2bfja5.pl
 zzz.com
 zzzmail.pl
-zzzzzzzzzzzzz.com
\ No newline at end of file
+zzzzzzzzzzzzz.com
```
Comment (lyrixx) [inline list.txt:55741]: Could you revert this change?

**Review by lyrixx — COMMENTED** [review]
Good, Except one little comment, 👍🏼 

Thanks

**Comment by FGRibreau** [comment]
Still waiting for https://github.com/FGRibreau/mailchecker/pull/431#discussion_r1521691474
closing this PR, please open another one if needed :)
